test/Microsoft.AspNet.Localization.FunctionalTests/TestRunner.cs

```diff
@@ -60,10 +65,52 @@ public class TestRunner
                 }
             }
         }
-
-        public string GetApplicationPath()
+        
+        public async Task RunTestAndVerifyResponseHeading(
```

Comment:
This method looks 99% the same as `RunTestAndVerifyResponse` - can more code be shared between them?